parsl/tests/test_execute_task.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+import pytest
+
+from parsl.executors.execute_task import execute_task
+from parsl.serialize.facade import pack_res_spec_apply_message
+
+
+def addemup(*args: int, name: str = "apples"):
+    total = sum(args)
```

Comment:
Pedantically, I observe that `int` is not iterable.

Reply:
my interpretation of https://typing.readthedocs.io/en/latest/spec/callables.html#annotating-args-and-kwargs is that `int` here means that this function takes an arbitrary number of positional parameters of type `int`.

Reply:
Ah; I've been educated.  Thank you; I wasn't aware of that syntax.  "Cheers to visual analysis," as opposed to _trusting the tools_.  :facepalm: